tensorflow/python/ops/image_ops_impl.py

```diff
@@ -774,9 +774,19 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
   This op does nothing if `offset_*` is zero and the image already has size
   `target_height` by `target_width`.
 
+  Usage example:
+
+    >>> x=tf.constant([
+    ...        [[1,2,3],[4,5,6],[7,8,9]],
+    ...        [[1,2,3],[4,5,6],[7,8,9]],
+    ...        [[1,2,3],[4,5,6],[7,8,9]]
+    ...        ])
+    >>> tf.image.pad_to_bounding_box(x, 28, 28, 100, 100)
```

Comment:
Change the bounding box to smaller values so that we could print all the values in `numpy=array([[[...`?

Reply:
got it. I'll reduce the size of both the arrays.